Lib/ufo2ft/featureCompiler.py

```diff
@@ -31,16 +31,16 @@ def parseLayoutFeatures(font):
     if not featxt:
         return ast.FeatureFile()
     buf = UnicodeIO(featxt)
-    # the path is used by the lexer to resolve 'include' statements
-    # and print filename in error messages. For the UFO spec, this
-    # should be the path of the UFO, not the inner features.fea:
-    # https://github.com/unified-font-object/ufo-spec/issues/55
     ufoPath = font.path
+    includeDir = None
     if ufoPath is not None:
-        buf.name = ufoPath
+        ufoPath = os.path.normpath(ufoPath)
+        # The `name` attribute ensures that error messages print the correct file name.
+        buf.name = os.path.join(ufoPath, "features.fea")
+        includeDir = os.path.dirname(ufoPath)
```

Comment:
I'd keep the comment that includes in UFO resolve relative to the parent directory of the UFO and the link to ufo-spec here, even only for historical reasons